common/functions/src/scalars/others/type_of.rs

```diff
@@ -61,7 +59,8 @@ impl Function for TypeOfFunction {
         columns: &common_datavalues::ColumnsWithField,
         input_rows: usize,
     ) -> Result<common_datavalues::ColumnRef> {
-        let type_name = format!("{:?}", columns[0].data_type());
+        let non_null_type = remove_nullable(columns[0].data_type());
+        let type_name = format_data_type_sql(&non_null_type);
```

Comment:
Shall we return the nullable sign?

Eg: `Nullable(VARCHAR)`

If yes, we should we `columns[0].data_type().sql_name()`.

Reply:
Only type name is better here?
I think the user don't understand what's `Nullable(VARCHAR)` mean :thinking:

Reply:
Ok